lib/kubernetes-deploy.rb

```diff
@@ -23,7 +11,34 @@
 require 'active_support/core_ext/hash/slice'
 require 'active_support/core_ext/numeric/time'
 
-class KubernetesDeploy
+module KubernetesDeploy
+  def self.logger=(value)
+    @logger = value
+  end
+
+  def self.logger
+    @logger ||= begin
+      l = Logger.new(STDOUT)
+      l.level = ENV["DEBUG"] ? Logger::DEBUG : Logger::INFO
+      l.formatter = proc do |severity, _datetime, _progname, msg|
+        case severity
+        when "FATAL", "ERROR" then "\033[0;31m[#{severity}]\t#{msg}\x1b[0m\n" # red
+        when "WARN" then "\033[0;33m[#{severity}]\t#{msg}\x1b[0m\n" # yellow
+        when "INFO" then "\033[0;36m#{msg}\x1b[0m\n" # blue
```

Comment:
This is the initial code from the script and I've been only moving it around to separate CLI from tests. 
Long term, we should use appropriate library for coloured output that would detect terminal support. I'd leave this to another PRs.